### DISCUSSION TITLE
[Support] <wg1 subnet and pivpn>

### DISCUSSION THREAD

**Discussion by leeuwtjex** [pivpn/pivpn#1378] [Q&A]
Hi team,

There is no issue here, meanwhile since my first ticket wireguard and pivpn run fine.
However, since I managed to configure a basic install, I would like to add a subnet,
to be able to separate client groups and prevent 'client to client' LAN activity, not all users should be able to visit other clients networks. 
I managed to configure a subnet  wg1  with 10.2.0.0  same as wg0 10.6.0.0.
Same firewall rules, same config, just another port and subnet.
But Unfortunately,  when i do #wg to list all wireguard interfaces, I see wg0 with all peers, but wg1 has no peers at all. I tried a lot of 'wg set' and 'wg-quick' commands to have the peer listed on wg1, 
but without success. it is just not listing the peer I tried to add.

Even if i would manage, it would stay a manual procedure to create every single new  user peer .conf file.
Why does pivpn support only a default wg0 subnet?  It would be very handy to build client peer
conf files for other subnets on the wireguard server.

Or is there another way to approach?

**Comment by coolapso** [comment]
Hy @leeuwtjex 

Unfortunately I cannot help you much there as its a kind of setup i haven't tried myself, also this is out of scope of pivpn, and the reason why we don't support such setup is because there was never a need or request for it, also pivpn mission is to porvide to provide those with less technical knowledge a way to have a vpn server. this being said we keep it as simple as possible.

**Comment by coolapso** [comment]
Converting this to discussion.
